test/services/validation/ERSValidationConfigCSOP.scala

```diff
@@ -116,8 +118,24 @@ class CSOPOptionsRCLTest extends PlaySpec with ERSValidationCSOPRCLTestData with
       )
     }
 
-  }
+    "when colB is answered yes then remaining columns from C to I are mandatory and if entered invalid or missing data then it should throw error " in {
+      val row = Row(1,getWronglyEnteredCellData)
+      val resOpt: Option[List[ValidationError]] = validator.validateRow(row)
+      resOpt.withErrorsFromMessages.get.size mustBe 3
```

Comment:
Is it worth checking what the errors are? similar to the test above `"when colB is answered yes, colC is a mandatory field" in {...`